api/routes/api_countries.py

```diff
@@ -0,0 +1,36 @@
+from flask import Blueprint, request
+from service.api_logic.countries_logic import get_countries
+from database.session import SessionLocal
```

Comment:
API MUST NOT use database layer in any way. Never. It only communicates with BusinessLayer.

So this session should be removed from here